nodeJS/express-basics/Express-Lesson-3.md

```diff
@@ -18,4 +19,9 @@ By the end of this lesson, you should be able to do the following:
 1. Continue where we left off with part 4 of the [Express tutorial](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes)
 </div>
 
+### Knowledge Check
 
+- <a class='knowledge-check-link' href='#assignment'>How do you define a route function in Express?</a>
+- <a class='knowledge-check-link' href='#assignment'>Name four HTTP verbs a route might need to handle.</a>
+- <a class='knowledge-check-link' href='#assignment'>What is a route parameter, and what syntax is used to define one in a route handler?</a>
+- <a class='knowledge-check-link' href='#assignment'>What is a route-handler callback function commonly called?</a>
```

Comment:
```suggestion
- <a class='knowledge-check-link' href='https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes'>How do you define a route function in Express?</a>
- <a class='knowledge-check-link' href='https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes'>Name four HTTP verbs a route might need to handle.</a>
- <a class='knowledge-check-link' href='https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes'>What is a route parameter, and what syntax is used to define one in a route handler?</a>
- <a class='knowledge-check-link' href='https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes'>What is a route-handler callback function commonly called?</a>
```
Linking directly to the resource being referenced, rather than just to the assignment section,  is a better user experience for these IMO.